lib/commands/schedule.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Commands
+  class Schedule < Base
+    def call
+      bot.api.send_message(
+        chat_id: chat_id,
+        text: Wrappers::Schedule.call(schedules: ::Schedule.all),
```

Comment:
Why `Wrappers`? That sounds unclear.

What we do here is **rendering** the text. So, it should be called renderer.

And if we're talking about renderers, why not using `erb` and adding a `render` method?

```ruby
bot.api.send_message(
  text: render(
    "schedule", # names of the template (could be inferred from the service name)
    schedules: ::Schedule.all # ERB template locals
   )
)
```